### PR TITLE
More t is now a parameter test

### PR DESCRIPTION
**Pull request by TorkelE**
Primarily in the test files. These were the last cases I could find, but there might possibly be more that I missed. Need to read through tomorrow as well to check it looks OK.

### CODE REVIEW

File: test/dsl/dsl_options.jl
Comment (TorkelE) [inline test/dsl/dsl_options.jl:439]: Generally I did not merge the new independent-variable-paraemter declarations with old parameter declarations. It seems desirable to create a `@independent_variables` declaration or similar for these in the future, so better keep them separate for when that happens.

File: test/reactionsystem_core/coupled_equation_crn_systems.jl
Comment (TorkelE) [inline test/reactionsystem_core/coupled_equation_crn_systems.jl:975]: This causes a a `@test_throws` to go from throwing an error to not throwing an error (hence causing test failure). The reason is not directly obvious, but should be possible to figure out since I've been able to look closer at it.

File: test/dsl/dsl_options.jl
Comment (isaacsas) [reply test/dsl/dsl_options.jl:439]: Sounds like that macro will happen, so we can update for it once available. For now we just want to ensure we don't get test failures or such from the change (and even if they change it to just be a warning now it would be good to avoid warnings in the test logs).

File: test/reactionsystem_core/coupled_equation_crn_systems.jl
Comment (isaacsas) [reply test/reactionsystem_core/coupled_equation_crn_systems.jl:975]: Interesting.

**Review by isaacsas — APPROVED** [review]
Feel free to merge when you are happy (and I guess make a 14.0.2 release to get the docs updated).

**Comment by isaacsas** [comment]
Approved conditional on the test failure getting fixed that is...

**Pull request merged**
